rust/experimental/lang/src/language/rust.rs

```diff
@@ -38,56 +54,107 @@ pub enum StateEl {
     // generics etc
 }
 
-pub fn to_style(style: &mut Style, el: &StateEl) {
-    match *el {
-        StateEl::Comment => style.fg_color = 0xFF75715E,
-        StateEl::StrQuote => style.fg_color = 0xFF998844,
-        StateEl::CharQuote => style.fg_color = 0xFF998844,
-        StateEl::Invalid => style.fg_color = 0xFFFF0000,
-        StateEl::NumericLiteral => style.fg_color = 0xFF6644EE,
-        StateEl::CharConst => style.fg_color = 0xFF8866EE,
-        StateEl::Keyword => style.font = 1,
-        StateEl::Operator => style.fg_color = 0xFFAA2244,
-        StateEl::PrimType => style.fg_color = 0xFF44AAAA,
+impl StateEl {
+    /// Transform a [StateEl] into a [Vec] of Syntect Scope [String]s.
```

Comment:
out of date?

Reply:
Yeah, this can go, but I'm going to keep the scope reference link around-- probably attach it to `ALL_SCOPES`